xija/gui_fit.py

```diff
@@ -16,23 +17,37 @@
 import json
 import logging
 
+from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
 from matplotlib.figure import Figure
-from matplotlib.backends.backend_gtkagg import FigureCanvasGTKAgg as FigureCanvas
-from matplotlib.backends.backend_gtkagg import NavigationToolbar2GTKAgg as NavigationToolbar
 
 import numpy as np
-import sherpa.ui as ui
+
 from Chandra.Time import DateTime
 import pyyaks.context as pyc
+import pyyaks.logger
 
 import Chandra.taco
 import xija
-import xija.clogging as clogging   # get rid of this or something
+import sherpa.ui as ui
+
+logging.basicConfig(level=logging.DEBUG)
+logging.debug('Importing gui_fit from {}'.format(__file__))
+
+# Globals for fitting, set in main
```

Comment:
@taldcroft is there a reason you had to switch to using `global`s here? they are not playing nice with the `multiprocessing` module.

Reply:
@jzuhone - Humm, I don't remember, sorry.  I browsed through the diffs and nothing rang a bell.  I'm sure there was a reason since otherwise I would not choose to replace working and better-structured code with globals like this...

So who knows if this branch is of much value or if you are better off just starting over from the original Gtk and just using this as a reference.